.github/workflows/release_build.yaml

```diff
@@ -58,18 +58,23 @@ jobs:
         uses: docker/setup-qemu-action@v3
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3
-      - name: Build artifact
+      - name: Build artifact (x86_64)
         run: make build
+      - name: Compress artifact (x86_64)
+        run: tar -czvf spiffe-helper_${{ github.ref_name }}_Linux-x86_64.tar.gz spiffe-helper
+      - name: Build artifact (arm64)
+        run: GOARCH=arm64 make build
```

Comment:
Should there be some kind of make clean target here or does it overwrite everything that could be generated in the previous step?